components/dashboard/type.go

```diff
@@ -151,9 +151,9 @@ type ExplorerBlock struct {
 	LikedInsteadChildren []string `json:"shallowLikeChildren"`
 	// Solid defines the solid status of the block.
 	Solid                  bool     `json:"solid"`
-	ConflictIDs            []string `json:"conflictIDs"`
-	AddedConflictIDs       []string `json:"addedConflictIDs"`
-	SubtractedConflictIDs  []string `json:"subtractedConflictIDs"`
+	SpendIDs               []string `json:"spendIDs"`
+	AddedSpendIDs          []string `json:"addedSpendIDs"`
```

Comment:
Make sure that the dashboard still works to the same extent as it did before and does not crash on the main page.